Makefile

```diff
@@ -0,0 +1,46 @@
+.PHONY: clean install
+
+clean:
+	rm -rf nighresjava/
+	rm -rf cbstools-public/
+	rm -rf imcn-imaging/
+	rm -rf nighres.egg-info/
+	rm -rf nighres_examples/
+
+install:
+	./build.sh
+	pip install .
+
+# =============================================================================
+# Docker related content
+
+.PHONY: Dockerfile docker_build
+
+Dockerfile:
+	docker run --rm repronim/neurodocker:0.7.0 generate docker \
+		--base debian:stretch-slim \
+		--pkg-manager apt \
+		--install "openjdk-8-jdk git wget build-essential software-properties-common libffi-dev" \
+		--copy conda-nighres.yml /tmp/conda-nighres.yml \
+		--miniconda \
+			create_env="nighres" \
+			conda_install="python=3.9 pip jcc gcc_linux-64 gxx_linux-64" \
+			activate="true" \
+		--env JAVA_HOME="/docker-java-home" \
+		--env JCC_JDK="/docker-java-home" \
+		--run 'ln -svT "/usr/lib/jvm/java-8-openjdk-$$(dpkg --print-architecture)" /docker-java-home' \
+		--copy Makefile build.sh cbstools-lib-files.sh setup.py MANIFEST.in README.rst LICENSE imcntk-lib-files.sh /home/neuro/nighres/ \
+		--copy nighres /home/neuro/nighres/nighres \
+		--workdir /home/neuro/nighres \
+		--run "conda init && . /root/.bashrc && conda activate nighres && conda info --envs && ./build.sh" \
+		--miniconda \
+			use_env="nighres" \
+			conda_install="jupyterlab Nilearn" \
+			pip_install="." \
+		--copy docker/jupyter_notebook_config.py /etc/jupyter \
+		--expose 8888 \
+		--cmd "jupyter lab --port=8888 --no-browser --ip=0.0.0.0" \
+		--user neuro > Dockerfile
```

Comment:
The whole dockerfile is generated using this neurodocker recipe.

One of the tricky bit when using conda within docker is this part: 

```
conda init && . /root/.bashrc && conda activate nighres && conda info --envs && ./build.sh
```

It requires to update the `.bashrc` of root and run it to be able to use conda in the rest of the `RUN` call, activate the  conda environment and then build all the nighres dependency

Reply:
Also there is no way that I know of to use the `conda-nighres.yml` file to set up the environment in docker with neurodocker the list of dependencies have to be "hard coded" in the make file at the moment.